app/models/communication/website/post.rb

```diff
@@ -31,9 +31,11 @@ class Communication::Website::Post < ApplicationRecord
   include WithMenuItemTarget
   include WithUniversity
 
-  belongs_to :author,
-             class_name: 'University::Person',
-             optional: true
+  has_and_belongs_to_many :authors,
+                          class_name: 'University::Person',
+                          join_table: :communication_website_posts_university_persons,
```

Comment:
Si tu changes le fichier de migration pour avoir "people" au lieu de "persons"

```suggestion
                          join_table: :communication_website_posts_university_people,
```